weepickle/src/main/scala/com/rallyhealth/weepickle/v1/LowPriorityImplicits.scala

```diff
@@ -17,19 +19,27 @@ abstract class LowPriorityImplicits
 
   private def toEnumerationName[E <: scala.Enumeration](e: E): To[E#Value] = {
     val cache = new ConcurrentHashMap[String, E#Value] // mitigate withName() slowness.
+    var lastVset: E#ValueSet = null
 
     ToString.map { s =>
-      var enum = cache.get(s) // 68x faster than withName()
+      val vset = e.values // exploit that e.values returns a cached instance unless new values are added
+      if (vset ne lastVset) {
```

Comment:
... my initial reaction to this code was, "Wait -- Enumeration isn't immutable?!?"  Then I read the documentation.  Yet another reason for me to be unfond of `scala.Enumeration`.